public_html/administrator/components/com_jed/views/review/tmpl/edit.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * @package    JED
+ *
+ * @copyright  Copyright (C) 2005 - 2020 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+
+/** @var JedViewReview $this */
+
+HTMLHelper::_('behavior.formvalidator');
+HTMLHelper::_('formbehavior.chosen', 'advancedSelect');
```

Comment:
The `advancedSelect` needs to be removed here, that is why the color won't show up.